lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
This doesn't explain why `mo.db.get()` has to be called here when it will give the exact same document as is already in the `doc` variable - except for `doc._rev` needing to be updated, which should have been captured when `insert()` was called.